roles/gluster_volume_create/defaults/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+# defaults file for gluster-volume-create
+volname: glustervol
```

Comment:
In the documentation above the volume name was gluster_cluster_volume, this will fail. Can you keep these uniform?